js/plugins/pluginPrototype.js

```diff
@@ -0,0 +1,15 @@
+function clinicalTimelinePlugin(name, spec){
```

Comment:
I think the spec is useful here, but I don't really understand why you need `name` and also `id` in some other plugins.